src/devices/models/deerma-jsq4.ts

```diff
@@ -0,0 +1,103 @@
+import type * as hb from "homebridge";
+import * as miio from "miio-api";
+import { MiotProtocol, MiotArg } from "../protocols";
+import { DeviceOptions } from "../../platform";
+import { ValueOf } from "../utils";
+import { Features } from "../features";
+import { HumidifierConfig } from ".";
+
+enum Mode {
+  Level1 = 1,
+  Level2 = 2,
+  Humidity = 3
+}
+
+type Props = {
+  power: boolean;
+  fan_level: Mode;
+  target_humidity: number;
+  water_level: number;
+  relative_humidity: number;
+  switch_status: boolean;
+  buzzer: boolean;
```

Comment:
```suggestion
  buzzer: boolean;
  temperature: number;
```

Reply:
I found on [miot-spec.org](http://miot-spec.org/miot-spec-v2/instance?type=urn:miot-spec-v2:device:humidifier:0000A00E:deerma-jsq4:1) that this model also supports `temperature`, so I think it also can be added